### PR TITLE
Add user create and delete

### PR DESCRIPTION
**Pull request by KeisukeYamashita**
## What

* Add these commands
    * `pd user create`
    * `pd user delete`

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,12 @@
+module github.com/PagerDuty/go-pagerduty
+
+require (
+	github.com/google/go-querystring v1.0.0
+	github.com/hashicorp/consul v1.6.1
```
[the remark this answers is not in the document]
Comment (KeisukeYamashita) [reply go.mod]: Sorry, I deleted it.

**Comment by theckman** [comment]
@KeisukeYamashita would you be able to refresh this PR so that I may review it? Some of the changes in here are already present in `master`, and so the refresh should help reduce the number of files to be reviewed.

**Comment by theckman** [comment]
@KeisukeYamashita similar to the other PR you raised, would you be able to rebase your changes against `master`? 

If we don't hear from you by the end of March, or you're not interested in moving it forward, we may look at closing this PR

**Comment by theckman** [comment]
Similar to the other PR, if you'd like to help gets this merged in, feel free to open a new PR. :+1: I'll close this one since it doesn't seem like there is that interest moving it forward right now

**Pull request closed without merging**